src/serializers/GltfSerializer.cpp

```diff
@@ -325,8 +336,12 @@ void GltfSerializer::finalize() {
 	json scene_0;
 	scene_0["nodes"] = node_array_;
 	json_["scenes"].push_back(scene_0);
-	json_["bufferViews"].push_back({ {"buffer", 0}, { "byteLength", indices_length } });
-	json_["bufferViews"].push_back({ {"buffer", 0}, {"byteStride", 12}, { "byteOffset", indices_length },  { "byteLength", binary_length - indices_length } });
+
+	for (auto &n : json_["bufferViews"]) {
```

Comment:
Could you add a comment to why this is necessary for the next reader. When the accessors are first written you don't know the size of the indices yet I guess...

Reply:
ok